src/lib/hooks.ts

```diff
@@ -46,7 +47,7 @@ function runTest(browser) {
       .run({
         skipJsErrors: true,
         selectorTimeout: TIMEOUT,
-        assertionTimeout: TIMEOUT,
+        assertionTimeout: 10000,
```

Comment:
must be TIMEOUT/2 or something like that https://devexpress.github.io/testcafe/documentation/using-testcafe/programming-interface/runner.html#run but never static o bigger than setDefaultTimeout

if you set 10000 always, when TIMEOUT is less than 10,000, it will fail